objectherkenning_openbare_ruimte/databricks_pipelines/common/databricks_workspace.py

```diff
@@ -0,0 +1,68 @@
+import json
+from datetime import datetime
+
+from databricks.sdk.runtime import *  # noqa: F403
+from pyspark.sql import SparkSession
+
+
+def get_databricks_environment(spark: SparkSession):
+    """
+    Returns Productie, Ontwikkel or None based on the tags set in the subscription
+    """
+
+    tags = spark.conf.get("spark.databricks.clusterUsageTags.clusterAllTags")
+    try:
+        tags_json = json.loads(tags)
+    except json.JSONDecodeError as e:
+        print(f"Error decoding JSON: {e}")
+        return None
+
+    environment_tag = next(
+        (tag for tag in tags_json if tag.get("key") == "environment"), None
+    )
+    if environment_tag:
+        environment = environment_tag.get("value")
+        return environment
+    else:
+        raise ValueError("Databricks environment is not set.")
+
+
+def get_catalog_name(spark: SparkSession):
+    """
+    Sets the catalog name based on the environment retrieved from Databricks cluster tags
+    """
+
+    environment = get_databricks_environment(spark)
+    if environment == "Ontwikkel":
+        catalog_name = "dpcv_dev"
+    elif environment == "Productie":
+        catalog_name = "dpcv_prd"
+
+    return catalog_name
```

Comment:
If it's neither of those you're returning an unexisting object, doesn't give an error?

Reply:
this i was supposed to delete altogether